
outdated? there's already in rpmio/Makefile.am:
AM_CFLAGS = $(OPENMP_CFLAGS)

--- rpm-5.4.18/rpmio/Makefile.am~	2017-07-16 22:47:06.000000000 +0300
+++ rpm-5.4.18/rpmio/Makefile.am	2017-07-18 16:16:56.558291408 +0300
@@ -274,7 +274,7 @@
 	yarn.c \
 	zstdio.c
 
-librpmio_la_LDFLAGS = -no-undefined -release $(LT_CURRENT).$(LT_REVISION)
+librpmio_la_LDFLAGS = -no-undefined -release $(LT_CURRENT).$(LT_REVISION) $(OPENMP_CFLAGS)
 if HAVE_LD_VERSION_SCRIPT
 librpmio_la_LDFLAGS += -Wl,@LD_VERSION_SCRIPT_FLAG@,@top_srcdir@/rpmio/librpmio.vers
 endif
--- rpm-5.3.1/tools/Makefile.am.wiget	2010-06-01 21:49:05.274449585 +0200
+++ rpm-5.3.1/tools/Makefile.am	2010-06-01 21:49:56.576943304 +0200
@@ -281,7 +281,7 @@ if WITH_DBSQL
 if WITH_DBSQL
 pkgbin_PROGRAMS +=	dbsql
 dbsql_SOURCES	= dbsql.c
-dbsql_LDFLAGS	= @LDFLAGS_STATIC@ $(LDFLAGS)
+dbsql_LDFLAGS	= @LDFLAGS_STATIC@ $(LDFLAGS) $(OPENMP_CFLAGS)
 dbsql_LDADD = \
 	$(RPMIO_LDADD_COMMON)
 endif
